tsconfig.json

```diff
@@ -24,7 +24,7 @@
     // linter checks
     "noImplicitReturns": false,
     "noFallthroughCasesInSwitch": true,
-    "noUnusedLocals": true,
+    // "noUnusedLocals": true,
```

Comment:
can we not use this. It means thinks won't build just because you temporarily comment out some code